src/sections/Partners/index.js

```diff
@@ -1,243 +1,163 @@
-          <Col xs={12} sm={10} lg={10}>
-            <div rightAlign={true} className="container">
-              <h2>The University of Texas at Austin</h2>
-              <div>
-                In partnership with faculty of the Cockrell School of Engineering at The University of Texas at Austin, graduate students engage in research on the affects of hardware architectures and the performance of software-based networking.
-                            Ph.D. candidates leverage Meshery in research and analysis of affect and overhead service mesh architectures have on when running distributed systems. Learn more about <a href="http://www.ece.utexas.edu/speakers/lee-calcote">our collaborative research</a>.
-              </div>
-            </div>
-          </Col>
-        </Row>
-        <Row>
-          <Col xs={12} sm={10} lg={10}>
-            <div rightAlign={true} className="container">
-              <h2>Google Summer of Code</h2>
-              <div>
-                Google Summer of Code is a global program focused on bringing more student developers into open source software development. Students work with an open source organization on a 3 month programming project during their break from school. Learn more about <Link to="/programs">our mentoring</Link>.
+        <div className="academic-container"> 
+          <div className="acad-container-body">
+            <h1 className="heading">Academic Partners</h1>
+            <p>Layer5, the service mesh community, established the Layer5 Partner Program to develop an ecosystem to advance higher learning of cloud native networking and to enable service providers to accelerate customers’ digital transformations, helping them move into the new era of DevOps and cloud native applications.</p>
+          </div>
+        </div>
+        {
+          academic_partners.map((partner,index) => (
+            <div key={index} className="acc-cont">
+              <Row  className={(index) % 2 === 1?"cont-row-reverse":"cont-row"} >
+                <Col xs={5} sm={3} lg={2} className="custom-col">
+                  <div className="img1">
+                    <a href={partner.imageRoute} target="_blank" rel="noopener noreferrer">
+                      <img  src={partner.imageLink} title="Click to know More about our partner" alt={partner.name} />
+                    </a>
+                  </div>
+                </Col>
+                <Col xs={12} sm={9} lg={10}>
+                  <div className="container">
+                    <h2>{partner.name}</h2>
+                    <div>
+                      {partner.desc}
+                    </div>
+                    {
+                      isOpen && ID === index? <button onClick={() => toggleActive(index)} className="readmeBtn"> Read Less <BsArrowUp className="icon" size={30} /></button> :
+                        <button onClick={() => toggleActive(index)} className="readmeBtn readmreBtn"> Read More <BsArrowDown className="icon" size={30} /></button> 
```

Comment:
I think the `readmreBtn` class is not used anywhere.

Reply:
Yes it is not used ...correct.!